src/app/home/webview-home/webview-home.component.scss

```
@@ -0,0 +1,39 @@
+    height: 100px;
+    display: flex; 
+    justify-content: center;
+        align-items: center;
+    transition: transform 0.3s ease-in-out;
+    img {
+        width: 100%;
+        height: 3rem;
+        flex: 1; 
+        object-fit: contain;
+    }
+    width: 100px;
```

Stylelint (17.3.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)